chainer/backends/cuda.py

```diff
@@ -637,3 +641,36 @@ def should_use_cudnn_tensor_core(dtype):
     if use_tensor_core is None:
         use_tensor_core = cudnn.is_tensor_core_available(dtype)
     return use_tensor_core
+
+
+# ------------------------------------------------------------------------------
+# cupy.cudnn utility
+# ------------------------------------------------------------------------------
+
+def get_cudnn_dropout_states():
+    if not cudnn_enabled:
+        raise RuntimeError('cuDNN is not enabled.')
+
+    thread_id = threading.get_ident()
```

Comment:
Please support Python2.7 . Python2.7 does not have `get_ident()`.
For example.
```
if hasatter(threading, 'get_ident'):
    thread_id = threading.get_ident()
else:
    thread_id = threading.current_thread().ident
```

Reply:
According to documents, python 2.7 and 3.x have almost same interfaces for `threading.Thread`.
We can also obtain this object by calling `threading.current_thread()`.

https://docs.python.org/3/library/threading.html#threading.Thread.ident
https://docs.python.org/2.7/library/threading.html#threading.Thread.ident

So I use the following code for obtaining a thread id in both Python 2.7 and 3.x:
```python
thread_id = threading.current_thread().ident
```